To-do list/Models/Task.swift

```diff
@@ -8,8 +8,14 @@
 import Foundation
 
 struct Task: Codable {
-    var title: String
-    var description: String
-    var deadline: String
-    var status: String
+    var title: String?
+    var description: String?
+    var deadline: Date?
+    var status: Status?
```

Comment:
why all your variable optional? In your case each variable represents a mandatory field.

Reply:
corrected, all  variables are mandatory now